exercicios/para-casa/entregas/karinne-almeida/cliente/interface/cliente.interface.ts

```diff
@@ -0,0 +1,16 @@
+export interface ClienteInterface {
+    nome: string,
+    endereco: endereco,
```

Comment:
💅🏾  
```suggestion
    endereco: Endereco,
```
lembra-se da convenção que para classes, enums, interfaces utilizar a convenção do PascalCase